# Remove preset CXX from EBSeq/Makefile, such that it uses the ones defined by EB.
# Author: Emik Lin (HKUMed CPOS)
--- RSEM-1.3.3.orig/EBSeq/Makefile	2020-02-15 03:21:41.000000000 +0800
+++ RSEM-1.3.3/EBSeq/Makefile	2025-10-27 11:19:00.636379917 +0800
@@ -1,4 +1,4 @@
-CXX = g++
+#CXX = g++
 PROGRAMS = EBSeq rsem-for-ebseq-calculate-clustering-info
 DEPENDENCIES = blockmodeling gplots gtools gdata caTools bitops KernSmooth
 
